### ISSUE TITLE
Comments inside interpolated expressions are not recognized

### ISSUE THREAD

**Issue by abacabadabacaba** [microsoft/qsharp-compiler#457]
This program should compile:
```
namespace Example {
    open Microsoft.Quantum.Intrinsic;
    @EntryPoint()
    operation Test() : Unit {
        let x = 1;
        Message($"{
            x
            // A comment
        }");
    }
}
```
But it doesn't. It compiles if the comment is removed.

**Comment by samarsha** [comment]
Thanks for the report! It does seem like it would make sense to support this. It looks like C# supports it as well.

**Comment by ScottCarda-MS** [comment]
This is resolved by #735.
